piq/functional/base.py

```diff
@@ -0,0 +1,56 @@
+from typing import Tuple
+import torch
+
+
+def ifftshift(x: torch.Tensor):
+    r""" Similar to np.fft.ifftshift but applies to PyTorch Tensors
+    """
+    shift = [-(ax // 2) for ax in x.size()]
+    return torch.roll(x, shift, tuple(range(len(shift))))
+
+
+def get_meshgrid(size: Tuple[int, int]) -> torch.Tensor:
+    r"""
+    Args:
+        size: Shape of meshgrid to create
```

Comment:
Wrong indentation here and following functions and files. 
```suggestion
        Args:
            size: Shape of meshgrid to create
```